contracts/truffle-config.js

```diff
@@ -5,9 +5,12 @@ const HDWalletProvider = require("@truffle/hdwallet-provider");
 module.exports = {
   networks: {
     development: {
-      host: "127.0.0.1",
-      port: 7545,
-      network_id: "*"
+      provider: () => new HDWalletProvider({
+        mnemonic: "two artist ribbon scene clown bachelor rail ivory grant budget clutch wrong",
```

Comment:
Is this mnemonic usable? (or, is it okay on reveal here?)

Reply:
Yes, I generated it for the test. I just hope others do not use the mnemonic code as his main wallet 😅